bin/console

```diff
@@ -3,6 +3,9 @@
 require 'bundler/setup'
 require 'fhir_client'
 
+# Default to INFO because Debug level makes the console hard to use
+FHIR.logger.level = Logger::INFO
```

Comment:
Could this get moved to the main part of the gem? Otherwise it makes the console difficult for any consumers of the gem to use.

Reply:
Updated, could you take a look to see if the new place is the best spot?